### DISCUSSION TITLE
Welcome to openSUSEway Discussions!

### DISCUSSION THREAD

**Discussion by denisok** [openSUSE/openSUSEway#59] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Albireo67** [comment]
Hi team !

I'm a regular openSUSE Tumbleweed user in my main system (with Plasma as DE), and I'm aiming to test openSUSEway in a different machine because I'd love to have TW running there as well with a tiling WM. 

I am not a developer (not even close!), just a normal user/hobbyist with limited scripting skills in Bash or Python, and I mean limited !

Anyway, I would like to share some ideas as I go along, any issues that I may encounter and not solve myself, and also test some tools for Sway that I've seen in other implementations and share feedback here. 

I hope that's an acceptable approach for you !

Thank you for the efforts you've made so far in this amazing project 😊 

Regards

**Reply by denisok** [reply]
sure, your feedback is Welcome!
